### ISSUE TITLE
Get stuck at epoch 1

### ISSUE THREAD

**Issue by miaojiang1987** [shtamura/glcic#2]
Dear Shtamura-san,

Thank you for helping me with the first issue. However, after I ran the train stage 1, I can finish Epoch 1 but the program just stop there. Could you help me point it out what happens?

My command is: python train.py --data_dir dataset --stage 1

Here is part of my log info:
2018-06-26 14:23:14,807 INFO #[139780687619840] None
2018-06-26 14:23:14,807 INFO #[139780687619840] 0 input_masked_image:False
2018-06-26 14:23:14,807 INFO #[139780687619840] 1 input_bin_mask:False
2018-06-26 14:23:14,807 INFO #[139780687619840] 2 lambda_1:True
2018-06-26 14:23:14,807 INFO #[139780687619840] 3 lambda_2:True
2018-06-26 14:23:14,807 INFO #[139780687619840] 4 lambda_3:True
2018-06-26 14:23:14,807 INFO #[139780687619840] 5 lambda_4:True
2018-06-26 14:23:14,807 INFO #[139780687619840] 6 generator:True
2018-06-26 14:23:14,807 INFO #[139780687619840] 7 concatenate_1:True
/home/ar-jiangmiao01/anaconda3/envs/glcic/lib/python3.5/site-packages/keras/engine/training.py:2039: UserWarning: Using a generator with 
 use_multiprocessing=True  and multiple workers may duplicate your data. Please consider using the keras.utils.Sequence class.
  UserWarning(Using a generator with use_multiprocessing=True)
Epoch 1/100
 99/100 [============================>.] - ETA: 0s - loss: 0.0233/home/ar-jiangmiao01/anaconda3/envs/glcic/lib/python3.5/site-packages/keras/engine/training.py:2197: UserWarning: Using a generator with  use_multiprocessing=True  and multiple workers may duplicate your data. Please consider using the`keras.utils.Sequence class.
  UserWarning('Using a generator with `use_multiprocessing=True`'

Then the program just stop there. Is it due to any other reason?

Best,
Miao

**Comment by shtamura** [comment]
Dear Miao-san

It means that 100 epoch can be completed, GPU is used. 
Since it thinks that it takes considerably 1 epoch if it is only CPU. 
If you are using GPU, I am sorry, I do not understand the cause. 
If you are using a CPU, please run it again in an environment that can use GPU.

Thank you.

**Comment by miaojiang1987** [comment]
Dear Shtamura-san,

Thank you for your quick answering. However, maybe I made you confused with my question.

Actually my problem is that after I finish running epoch 1, I can't move on to epoch 2. Is it because the progress of epoch 2 will not be showed up? Or I do have to wait some time?

Secondly, have you received a warning like this? 
/python3.6/site-packages/keras/engine/training.py:2087: UserWarning: Using a generator with 
 use_multiprocessing=True and multiple workers may duplicate your data. Please consider using the keras.utils.Sequence class.

Best,
Miao

**Comment by miaojiang1987** [comment]
Dear Shatamura-san,

Problem solved. Could you please add comment to the instructions? I find that the name of training and testing folder should be  dataset/train and dataset/val? I named it as dataset/testing so I got stuck.

However, I really appreciate your great work and thank you for your sharing. 

Best,
Miao
